registry/repositories/connection.postgres.ts

```diff
@@ -0,0 +1,10 @@
+import { Pool } from "pg";
+
+const PG_CONNECTION_STRING = process.env.PG_CONNECTION_STRING;
```

Comment:
Similar comment as above, idealy there would be some logic that throws an error on service start of the connection string env variable is not provided.